cmds/vpdbootmanager/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+const usage = `Usage:
+boot add [netboot [dhcpv6|dhcpv4] [MAC] | localboot [grub|path [Device GUID] [Kernel Path]]]
+
+Ex.
+boot add localboot grub
+
+boot add netboot dhcpv6 AA:BB:CC:DD:EE:FF
+
+Use -h to show flag options.
+`
```

Comment:
can you also add an option that just prints the JSON value without writing to VPD? It will help with non in-band changes